File: helloworld.php
<html>
 <head>
  <title>PHP Test</title>
 </head>
 <?php
  $bg = array('red', 'blue', 'white'); // array of filenames

  $i = rand(0, count($bg)-1); // generate random number size of the array
  $selectedBg = "$bg[$i]"; // set variable equal to which random color was chosen
?>
 <body style="background:<?php echo $selectedBg; ?>">
 <?php echo '<p>Hello Web World from PHP on' . date("Y/m/d") .' '. date("h:i:sa") . '</p>'; ?> 
 </body>
</html>

File: cgi-bin/helloworld.pl
#!/usr/bin/perl
# helloworld.pl - My first CGI program

print "Content-Type: text/html\n\n";
# Note there is a newline between 
# this header and Data

# Simple HTML code follows

print "<html> <head>\n";
print "<title>Hello world in Perl!</title>";
print "</head>\n";
@colors = ("blue", "white", "red");
$num = int rand(3);
$color =  $colors[$num];
print "<body style=\"background-color: $color \">\n";
$datestring = localtime();
print "<h1>Hello, world from $datestring!</h1>\n";
print "</body> </html>\n";
